FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -474,8 +627,9 @@ import Foundation
     private let auth: Auth
     private let callbackScheme: String
     private let usingClientIDScheme: Bool
+    private var recaptchaVerifier: AuthRecaptchaVerifier?
 
-    init(auth: Auth) {
+    init(auth: Auth, recaptchaVerifier: AuthRecaptchaVerifier? = nil) {
```

Comment:
nit: it doesn't look like this param is assigned to a property so I don't think it needs to be injected
```suggestion
    init(auth: Auth) {
```

Reply:
It feels a bit risky at this point to update this. A full e2e test should be conducted on this after the change.  I have created an internal tracking bug for this and we can address this asap.  b/381080410.

Reply:
Sorry, my original comment was not very clear. My understanding was that this initializer's parameter was doing nothing and was unused, because I don't see the parameter name referenced anywhere in the initializer's implementation.